website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -0,0 +1,255 @@
+
+TL/DR: For alerting current and impending disaster situations.  These alerts
+page an SRE.  The situation should warrant waking someone in the middle of the
+night.
+
+Timeline:  ~5 minutes.
+
+Reserve critical level alerts only for reporting conditions that may lead to
+loss of data or inability to deliver service for the cluster as a whole.
+Failures of most individual components should not trigger critical level alerts,
+unless they would result in either of those conditions. Configure critical level
+alerts so they fire before the situation becomes irrecoverable. Expect users to
+be notified of a critical alert within a short period of time after it fires so
+they can respond with corrective action quickly.
+
+Example critical alert: [KubeAPIDown](https://github.com/openshift/cluster-monitoring-operator/blob/79cdf68/assets/control-plane/prometheus-rule.yaml#L412-L421)
+
+```yaml
+- alert: KubeAPIDown
+  annotations:
+    summary: Target disappeared from Prometheus target discovery.
+    description: KubeAPI has disappeared from Prometheus target discovery.
+    runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-monitoring-operator/KubeAPIDown.md
+  expr: |
+    absent(up{job="apiserver"} == 1)
+  for: 15m
+  labels:
+    severity: critical
+```
+
+This alert fires if no Kubernetes API server instance has reported metrics
+successfully in the last 15 minutes.  This is a clear example of a critical
+control-plane issue that represents a threat to the operability of the cluster
+as a whole, and likely warrants paging someone.  The alert has clear summary and
+description annotations, and it links to a runbook with information on
+investigating and resolving the issue.
+
+The group of critical alerts should be small, very well defined, highly
+documented, polished and with a high bar set for entry.  This includes a
+mandatory review of a proposed critical alert by the Red Hat SRE team.
+
+##### Warning Alerts
+
+TL/DR: The vast majority of alerts should use the severity.  Issues at the
+warning level should be addressed in a timely manner, but don't pose an
+immediate threat to the operation of the cluster as a whole.
+
+Timeline:  ~60 minutes
+
+If your alert does not meet the criteria in "Critical Alerts" above, it belongs
+to the warning level or lower.
+
+Use warning level alerts for reporting conditions that may lead to inability to
+deliver individual features of the cluster, but not service for the cluster as a
+whole. Most alerts are likely to be warnings. Configure warning level alerts so
+that they do not fire until components have sufficient time to try to recover
+from the interruption automatically. Expect users to be notified of a warning,
+but for them not to respond with corrective action immediately.
+
+Example warning alert: [ClusterNotUpgradeable](https://github.com/openshift/cluster-version-operator/blob/513a2fc/install/0000_90_cluster-version-operator_02_servicemonitor.yaml#L68-L76)
```

Comment:
Have we any examples that do not come from OCP?

Reply:
I would like to replace these examples with memchached alerts, once we have them in place. That is the next item for Assaf to work on.

Reply:
@camilamacedo86 ^ woul it make sense to use memchached examples here?